syntax = "proto3";

package outer;

service OuterService {
  rpc TalkWithClient(stream TalkWithClientRequest) returns (stream TalkWithClientResponse) {}
}

message TalkWithClientRequest {
  string msg = 1;
}

message TalkWithClientResponse {
  string msg = 1;
}
